opentelemetry-sdk/src/opentelemetry/sdk/_metrics/aggregation.py

```diff
@@ -141,7 +141,7 @@ def collect(self) -> Optional[Gauge]:
 class ExplicitBucketHistogramAggregation(Aggregation[Histogram]):
     def __init__(
         self,
-        boundaries: Sequence[int] = (
+        boundaries: Sequence[float] = (
             0,
```

Comment:
To match the `float` type these values should be `0.0`, `5.0`... I think

Reply:
from a typing perspective, the ints are valid, but i can change it